src/detectors/LUMISPECCAL/CalorimeterHit_factory_EcalLumiSpecRecHits.h

```diff
@@ -0,0 +1,91 @@
+
+#pragma once
+
+
+#include <services/io/podio/JFactoryPodioT.h>
+
+#include <algorithms/calorimetry/CalorimeterHitReco.h>
+#include <services/log/Log_service.h>
+#include <extensions/spdlog/SpdlogExtensions.h>
+
+class CalorimeterHit_factory_EcalLumiSpecRecHits : public eicrecon::JFactoryPodioT<edm4eic::CalorimeterHit>, CalorimeterHitReco {
+
+public:
+    //------------------------------------------
+    // Constructor
+    CalorimeterHit_factory_EcalLumiSpecRecHits(){
+        SetTag("EcalLumiSpecRecHits");
+        m_log = japp->GetService<Log_service>()->logger(GetTag());
+    }
+
+    //------------------------------------------
+    // Init
+    void Init() override{
+        auto app = GetApplication();
+
+        m_input_tag = "EcalLumiSpecRawHits";
+
+        // digitization settings, must be consistent with digi class
+        m_capADC=16384;//{this, "capacityADC", 8096};
+        m_dyRangeADC=20. * dd4hep::GeV;//{this, "dynamicRangeADC", 100. * dd4hep::MeV};
+        m_pedMeanADC=100;//{this, "pedestalMean", 400};
+        m_pedSigmaADC=1;//{this, "pedestalSigma", 3.2};
+        m_resolutionTDC=10 * dd4hep::picosecond;//{this, "resolutionTDC", 10 * ps};
+
+        // zero suppression values
+        m_thresholdFactor=4.0;//{this, "thresholdFactor", 0.0};
+        m_thresholdValue=3.0;//{this, "thresholdValue", 0.0};
+
+        // energy correction with sampling fraction
+        m_sampFrac=0.998;//{this, "samplingFraction", 1.0};
```

Comment:
Value not known yet, I assume:
```suggestion
        m_sampFrac=1.0;
```

Reply:
Right, but I was just following the lead from other ECALs.  They seem (or seemed) to all have 0.998 as a default.

Reply:
I fully understand. We need to do this in a systematic way too. However, I'm looking to encourage detector experts to critically assess those values and then update/document them.